### PR TITLE
update operator-sdk to 1.8.0

### PR DESCRIPTION
**Pull request by ZhuoxiLi**

### CODE REVIEW

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml
Comment (horis233) [inline bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: Could we use a real image version, like
```
quay.io/opencloudio/common-service-operator@sha256:sha256:e7f42050d54530c708f162abcf04aae1b63ce6f5ca6e97af81a4a83f6ae708f6
ibm-common-service-operator.v3.8.0
```

Comment (ZhuoxiLi) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: sure, we could use this. The question is if we are using a specific version image, we will need to update the tag/sha every release. That will be some duplicated work. Any suggestions on that?

Comment (Daniel-Fan) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: Could we use tag instead of digest?

So it will be updated every time when we bump the version.

Comment (ZhuoxiLi) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: that sounds good to me.

Comment (horis233) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: We need to use digest here: https://docs.openshift.com/container-platform/4.7/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs

Is there any difference from using digest and tag? We still need to update it every release.

Comment (Daniel-Fan) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:332]: If we use digest here, we need to update it before closing to GA, after the image has been published on quay.io.
But it seems that we does not many choices

Comment (ZhuoxiLi) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml]: Updated with the digest, thank you all for the discussion here. Please review again,

Comment (horis233) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml]: I get what you mean. Then I believe we can hold this PR until we can get some help from the CICD

Comment (horis233) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml]: @ZhuoxiLi 
Please revert the changes relevant to related images and merge this PR

**Comment by ZhuoxiLi** [comment]
@horis233 @Daniel-Fan this pr now only contains the change of `update operator-sdk to 1.8.0`, after we merged this pr, please also upgrade your local operator-sdk to 1.8.0. thanks!

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**
